packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -464,7 +472,10 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
         call?.localVideoStreams.find((stream) => stream.mediaStreamType === 'Video') ||
         deviceManager?.getUnparentedVideoStreams().find((stream) => stream.mediaStreamType === 'Video');
       if (stream) {
-        return stream.feature(Features.VideoEffects).startEffects(new BackgroundBlurEffect(backgroundBlurConfig));
+        const createEffect =
```

Comment:
If someone call this handler function onBlurVideoBackground, and if there is no onResolveVideoBackGroundDependency , we should throw runtime errors